backends/advanced/src/advanced_omi_backend/services/memory/providers/openmemory_mcp.py

```
@@ -382,38 +385,38 @@ async def delete_memory(self, memory_id: str, user_id: Optional[str] = None, use
             await self.initialize()
-        
+
         # Update MCP client user context for this operation
         original_user_id = self.mcp_client.user_id
         self.mcp_client.user_id = self.user_id  # Use configured user ID
-        
+
         try:
             count = await self.mcp_client.delete_all_memories()
             memory_logger.info(f"🗑️ Deleted {count} memories for user {user_id} via OpenMemory MCP")
             return count
```

Ruff (0.14.14)

Consider moving this statement to an `else` block

(TRY300)